flink-ai-flow/ai_flow/metadata_store/service/service.py

```diff
@@ -407,3 +408,44 @@ def deleteArtifactById(self, request, context):
     def deleteArtifactByName(self, request, context):
         status = self.store.delete_artifact_by_name(request.name)
         return _wrap_delete_response(status)
+
+    @catch_exception
+    def registerWorkflow(self, request, context):
+        workflow = transform_workflow_meta(request.workflow)
+        response = self.store.register_workflow(name=workflow.name,
```

Comment:
Does this need to handle the field `properties` like `properties = None if request.properties == {} else request.properties`?

Reply:
Yes, it's handled in upper `transform_workflow_meta`